docs/en/academic/presentations.md

```diff
@@ -108,20 +108,56 @@ const zettlrRevealOptions = {
 
 If none of the available themes appeal to you, or if you need to provide some extra CSS, you can do so simply by dropping in a `<style>` tag somewhere in your presentation.
 
-You'll need some knowledge of CSS to finegrain the styling, but the appearance of the presentation itself should be satisfying on its own.
+You'll need some knowledge of CSS to fine tune the styling, but the appearance of the presentation itself should be acceptable without these tweaks.
 
 ![presentations_styles.png](../img/presentations_styles.png)
 
 ## Images
 
-One of the drawbacks of such HTML presentations is that images cannot be contained in the resulting file. (More accurately, they _can_ be included, but that would increase the file size by the size of all images multiplied by the factor 1.3.)
+One of the drawbacks of these HTML presentations is that images cannot be contained in the resulting file. (More accurately, they _can_ be included, but that would increase the file size by the size of all images multiplied by the factor 1.3.)
 
-So you need to decide where to put the images. There are basically three feasible methods.
+So you need to decide where to put the images. You have three options:
 
 1. Create a dedicated directory for the images, e.g., `img`, inside the directory with your markdown file. Then link to all images using relative paths (`img/my-image.jpg`). When you export your file, make sure the presentation is in the same place relative to the image directory as your source file.
 2. Simply put all images inside the same directory as your presentation file. Then linking to them using their file name should suffice.
-3. Upload them to the web, for instance unto an image hosting service, and link their URLs. Only drawback: You would need to rely on a working internet connection at the place where you hold your presentation.
+3. Upload them to the web, for instance into an image hosting service, and link their URLs. The drawback to this approach is that you are relying on a working internet connection to give your presentation.
 
 A rule of thumb is: If Zettlr correctly displays your images, so will your presentation, as long as you make sure the presentation is in _exactly_ the right place relative to your image files.
 
 > If you really must, you can tell Pandoc to include all images directly in the HTML file by setting the variable `self-contained` to `true` in your defaults file (it does not work inside the frontmatter). But remember that many images in your presentation could easily result in an HTML file of dozens of Megabyte. However, then you'd have a truly self-contained presentation file.
+
+## Workspaces and presentations
+
+If you create a new Markdown presentation file (File -> New File…) and save it to a new directory, you won't be able to render your presentation. Zettlr will inform you that 'The requested file was not found'. 
```

Comment:
Ah you mean when you use the file dialog to navigate to a different directory? If so that sounds like a bug! The file should be opened as a root file (and thus be found by the FSAL when exporting)

Reply:
**Description of how problem occurred**

* I was working on documentation. 
* Using the 'File' menu, I created a new file, added some basic text, with headers, to mimic a slide presentation.
* I saved the file; in the save pop-menu, I navigated to my Desktop, created a new folder called 'test', and saved the presentation in that folder.
* When I tried to render the present, I got `The requested file was not found` error.
* I had File -> Open Workspace...  and select the 'test' folder I had previously created.
* Now I could render the presentation properly.

Is this a bug or is it how Zettlr is suppose to work? I just replicated the error on a different computer, so it seems to be pretty reproducible. If it is a bug I will delete the highlighted section from the documentation. If it is not a bug, I will leave it in, but may add additional details?! 

Let me know what you think.

Reply:
It's a bug! So you can close this out, I'll make sure to add the file to the FSAL in any case.